.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
   - export PATH=$M2_HOME/bin:$PATH
 script:
   - |
```

Comment:
Sorry, my bad.  Usually I tend to consider the commits less important because it's usually a full feature, and it's just minor changes / fixes afterwards.

I'll try to make sure the messages are easier to follow, since this is pretty ongoing until it's consistent.